commit d039da1c00e01f8d3c3d74f439a971eb73e3045e
Author: H.J. Lu <hjl.tools@gmail.com>
Date:   Wed Jun 26 15:07:18 2019 -0700

    x86: Add sysdeps/x86/dl-lookupcfg.h
    
    Since sysdeps/i386/dl-lookupcfg.h and sysdeps/x86_64/dl-lookupcfg.h are
    identical, we can replace them with sysdeps/x86/dl-lookupcfg.h.
    
            * sysdeps/i386/dl-lookupcfg.h: Moved to ...
            * sysdeps/x86/dl-lookupcfg.h: Here.
            * sysdeps/x86_64/dl-lookupcfg.h: Removed.

Conflicts:
	sysdeps/x86_64/dl-lookupcfg.h
	  (Removal after copyright year update.)

diff --git a/sysdeps/i386/dl-lookupcfg.h b/sysdeps/x86/dl-lookupcfg.h
similarity index 100%
rename from sysdeps/i386/dl-lookupcfg.h
rename to sysdeps/x86/dl-lookupcfg.h
diff --git a/sysdeps/x86_64/dl-lookupcfg.h b/sysdeps/x86_64/dl-lookupcfg.h
deleted file mode 100644
index 5399cf25abde592d..0000000000000000
--- a/sysdeps/x86_64/dl-lookupcfg.h
+++ /dev/null
@@ -1,31 +0,0 @@
-/* Configuration of lookup functions.
-   Copyright (C) 2005-2018 Free Software Foundation, Inc.
-   This file is part of the GNU C Library.
-
-   The GNU C Library is free software; you can redistribute it and/or
-   modify it under the terms of the GNU Lesser General Public
-   License as published by the Free Software Foundation; either
-   version 2.1 of the License, or (at your option) any later version.
-
-   The GNU C Library is distributed in the hope that it will be useful,
-   but WITHOUT ANY WARRANTY; without even the implied warranty of
-   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-   Lesser General Public License for more details.
-
-   You should have received a copy of the GNU Lesser General Public
-   License along with the GNU C Library; if not, see
-   <http://www.gnu.org/licenses/>.  */
-
-#define DL_UNMAP_IS_SPECIAL
-
-#include_next <dl-lookupcfg.h>
-
-/* Address of protected data defined in the shared library may be
-   external due to copy relocation.   */
-#define DL_EXTERN_PROTECTED_DATA
-
-struct link_map;
-
-extern void _dl_unmap (struct link_map *map) attribute_hidden;
-
-#define DL_UNMAP(map) _dl_unmap (map)
